pkg/scheduler/framework/plugins/defaultpreemption/default_preemption.go

```diff
@@ -169,7 +222,7 @@ func (pl *DefaultPreemption) preempt(ctx context.Context, state *framework.Cycle
 // Each candidate is executable to make the given <pod> schedulable.
 func FindCandidates(ctx context.Context, cs kubernetes.Interface,
 	state *framework.CycleState, pod *v1.Pod, m framework.NodeToStatusMap, ph framework.PreemptHandle,
-	pl *DefaultPreemption, nodeLister framework.NodeInfoLister, pdbLister policylisters.PodDisruptionBudgetLister) ([]Candidate, error) {
+	nodeLister framework.NodeInfoLister, pdbLister policylisters.PodDisruptionBudgetLister, opts ...interface{}) ([]Candidate, error) {
```

Comment:
create a type for the options. See https://github.com/kubernetes/kubernetes/blob/90be932b600dd277601614840fbe510161fc813d/pkg/scheduler/scheduler.go#L183

Reply:
Done.